lib/Cavil/Bootstrap.pm

```diff
@@ -28,27 +28,30 @@ sub load {
   # Avoid adding the same data more than once
   my $licenses = $self->app->licenses;
   return undef
-    if $licenses->pg->db->select('licenses', 'id', {name => 'Low Risk Keyword'})
+    if $licenses->pg->db->select('licenses', 'id', {name => 'Apache-2.0'})
```

Comment:
Was this change intentional?

Reply:
Yes. there is no license 'low risk keyword' anymore.